test/integration/crcsuite/crcsuite.go

```diff
@@ -247,6 +249,48 @@ func CheckOutputMatchWithRetry(retryCount int, retryTime string, command string,
 	return matchErr
 }
 
+// CheckCRCStatus checks that output of status command
+// matches given regex number of consecutive times
+func CheckCRCStatus(state string) error {
+
+	retryDuration, _ := time.ParseDuration("1m")
+	retryCount := 15
+	cmd := "crc status --log-level debug"
+	expression := ``
+	var numConsecutive int
+
+	if state == "Stopped" {
+		numConsecutive = 1
+		expression = `.*Stopped.*`
+	} else {
+		numConsecutive = 3
```

Comment:
So this number is present to have consecutive execute the crc status and see if have same output?

Reply:
Yes. Exactly. In particular, I assume that if OCP cluster has been Running for 3mins in a row, then it's "stable" for the rest of the Feature (in particular, this check avoids "Running -> Not reachable -> Starting" pitfall right after start.